test/unit/hubAuth.test.ts

```diff
@@ -9,6 +9,8 @@ describe('hubAuth', () => {
   it('works with existing defaultdevhub if provided');
   it('auths from authurl in env');
   it('auths from authurl and sets as default');
+  it('auths from access token and sets as default');
+  it('auths from access token');
```

Comment:
Are we going to ignore these unit tests forever?  Even having NUTs for this would be something.  Right now there's nothing.